embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/injection/OpenTelemetryModuleImpl.kt

```diff
@@ -31,12 +32,7 @@ internal class OpenTelemetryModuleImpl(
     )
 ) : OpenTelemetryModule {
 
-    init {
-        if (!initModule.useKotlinSdk) {
-            // Enforce the use of default OTel Java SDK ThreadLocal ContextStorage to bypass SPI looking that violates Android strict mode
-            System.setProperty("io.opentelemetry.context.contextStorageProvider", "default")
-        }
-    }
+    private var useKotlinSdk = DEFAULT_USE_KOTLIN_SDK
```

Comment:
Can `DEFAULT_USE_KOTLIN_SDK` be confined to a private constant within `OtelBehavior`?